### ISSUE TITLE
Is there any ways to modify "Choose a file" image upload button click event handler?

### ISSUE THREAD

**Issue by opdev1004** [nhn/tui.editor#2141]
## Summary
Good day.

Is there any ways to modify "Choose a file" image upload button click event handler?

I am trying to put my custom event handler for the "Choose a file" button of image uploading.

I know there's "addImageBlobHook" but this one happens after image file has been choosen from the open file dialogue.

## Screenshots
None.

## Version
3.1.1

## Additional context
None.

**Comment by js87zz** [comment]
@opdev1004 
Currently, that functionality is not provided.
For what purpose are you adding it?

**Comment by opdev1004** [comment]
@js87zz
Oh, it is my bad. I should have added more information in the additional context section.

My project is using the toast ui editor within webview for the desktop application.

And one of the feedback that I got from my app user was, they want the absolute/relative image file path instead of base64 encoded image.

Then I was thinking to modify the "Choose a file" image upload button event handler.

So the button can call open file dialogue from the file system directly instead of using the one from webview/webbrowser.

**Comment by deezaster** [comment]
Hey @opdev1004 

That's what my customers want, too. For images, allow links only, no encoded images.

It would be enough for me if the "Insert Image" dialog is customizable.

**Comment by opdev1004** [comment]
@deezaster That's good 😃

Just for the people who want other example for the custom button:
https://github.com/nhn/tui.editor/issues/1204#issuecomment-931750895

**Comment by deezaster** [comment]
My dear friend @opdev1004. **Why are you only now coming up with this link**? Anyway, now I also have a solution.

Thanks for your active help! 

Nevertheless, a detailed and clear documentation would be very helpful.

**Comment by deezaster** [comment]
I have added 2 custom buttons. Now this:
#2028

**Comment by opdev1004** [comment]
I don't think this editor provide that feature. So i would put an element that can attach onload event.

https://stackoverflow.com/questions/4057236/how-to-add-onload-event-to-a-div-element#answer-41176554

**Comment by RenukaCheripally** [comment]
I have a similar issue. I need to restrict the "Choose a file" image button to accept only specific set of files. Do we have a fix in the editor yet?
